### ISSUE TITLE
poet init does not work

### ISSUE THREAD

**Issue by kootenpv** [sdispater/poet#2]
Installed latest from pip just now.

```bash
pascal@MBP:~/egoroot/newnew $ poet init


  Welcome to the Poet config generator



This command will guide you through creating your poetry.toml config.



  [FileNotFoundError]
  [Errno 2] No such file or directory: '/Users/pascal/newnew/poetry.toml'


init [-i|--index INDEX] [--name NAME] [--description DESCRIPTION] [--author AUTHOR] [--dependency DEPENDENCY] [--dev-dependency DEV-DEPENDENCY] [-l|--license LICENSE]
```

**Comment by sdispater** [comment]
Thanks for reporting this. I will look into it.

**Comment by kootenpv** [comment]
python 3.5.0 on OSX in case you're wondering.

As an aside, I was expecting this to work:

    import poet
    poet.__version__

but that just might be a personal preference.

**Comment by sdispater** [comment]
Can you launch the command with the `-v` option to get the full stack trace?

**Comment by kootenpv** [comment]
```bash
pascal@MBP:~/egoroot/newnew $ poet init -v


  Welcome to the Poet config generator



This command will guide you through creating your poetry.toml config.



  [FileNotFoundError]
  [Errno 2] No such file or directory: '/Users/pascal/newnew/poetry.toml'


Exception trace:
 /Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/cleo/application.py in run() at line 96
   status_code = self.do_run(input_, output_)
 /Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/cleo/application.py in do_run() at line 188
   status_code = command.run(input_, output_)
 /Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/cleo/commands/command.py in run() at line 90
   return super(Command, self).run(i, o)
 /Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/cleo/commands/base_command.py in run() at line 144
   status_code = self.execute(input_, output_)
 /Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/poet/console/commands/command.py in execute() at line 74
   return super(Command, self).execute(i, o)
 /Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/cleo/commands/command.py in execute() at line 99
   return self.handle()
 /Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/poet/console/commands/init.py in handle() at line 59
   git_config = self.git_config()
 /Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/poet/console/commands/init.py in git_config() at line 317
   return self.poet.git_config
 /Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/poet/console/commands/command.py in poet() at line 35
   self._poet = Poet(self.poet_file)
 /Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/poet/poet.py in __init__() at line 51
   with open(self._path) as f:

init [-i|--index INDEX] [--name NAME] [--description DESCRIPTION] [--author AUTHOR] [--dependency DEPENDENCY] [--dev-dependency DEV-DEPENDENCY] [-l|--license LICENSE]
```

**Comment by sdispater** [comment]
I just released the `0.3.2` version which fixes the `init` command.

**Comment by kootenpv** [comment]
Thanks, that solved it.
